templates/chartmuseum/chartmuseum-sm.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.chartmuseum.enabled .Values.chartmuseum.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: "{{ template "harbor.chartmuseum" . }}"
+  labels:
+{{ include "harbor.labels" . | indent 4 }}
+    component: chartmuseum
+spec:
+  endpoints:
+    - interval: 10s
```

Comment:
This should be configurable or left blank so that prometheus can use the server default value

Reply:
don't care